app/view_models/admin/draft_list_item.rb

```diff
@@ -15,7 +15,7 @@ def id
 
   def c2_proposal_status
     if auction.purchase_card == 'default'
-      c2_proposal_status_presenter.status
+      I18n.t("drafts.c2_status.#{auction.c2_status}.status")
```

Comment:
I am not sure why -- I think that using the C2 presenters made sense when all we had was C2 presenters. Also, I believe this was implemented before we were using I18n, so interpolation would not have been an option since we were just returning strings.